routes/refreshToken.routes.js

```diff
@@ -0,0 +1,7 @@
+const express = require("express");
+const router = express.Router();
+const authController = require("../controllers/auth");
+
+router.get("/", authController.refreshTokens);
```

Comment:
refresh token should be handled through middleware, not as a separate route

Reply:
removed that rout